code/atom.dm

```diff
@@ -640,45 +640,38 @@
 /atom/proc/special_desc(dist, mob/user)
 	return null
 
-/atom/proc/examine()
-	set name = "Examine"
-	set category = "Local"
-	set src in view(12)	//make it work from farther away
-
-	if(src.hiddenFrom && hiddenFrom.Find(usr.client)) //invislist
+/atom/proc/examine(mob/user)
```

Comment:
The change in signature is a code hygiene thing, this allows things that run on, for example, the server host `usr` scope to trigger examines.